html/terminal.css

```
@@ -0,0 +1,117 @@
+  --vev-color-10: #00ff00;
+  --vev-color-11: #ffff00;
+  --vev-color-12: #0000ff;
+  --vev-color-13: #ff00ff;
+  --vev-color-14: #00ffff;
+  --vev-color-15: #ffffff;
+  --vev-font-family: ui-monospace, "Cascadia Mono", "Noto Sans Mono", monospace;
+  --vev-font-size: 14px;
+  --vev-line-height: 1.2;
+  --vev-cell-width: 1ch;
+  --vev-cell-height: calc(var(--vev-font-size) * var(--vev-line-height));
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)